content/documentation/v16/getting-started.md

```diff
@@ -28,7 +28,7 @@ Dependency:
 {{< highlight groovy "linenos=table" >}}
 
     dependencies {
-      compile 'com.graphql-java:graphql-java:15.0'
```

Comment:
I had raised [this PR](https://github.com/graphql-java/graphql-java-page/pull/42) a week back to fix the same 🙂  Usually, I haven't seen minor number specific version mentioned in the getting-started.md file.

Reply:
Understood, but 16.0 was the exception where the released library is corrupt as well documented in the github release notes

Reply:
👍🏻